app/routes/admin/settings/index.js

```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.l10n.t('System');
+  },
+  model() {
```

Comment:
I don't think this is needed. The model would anyway be inherited from the parent if there is no model method in the child.